spec.emu

```diff
@@ -495,18 +697,27 @@ contributors: Ujjwal Sharma, Younies Mahmoud
         1. Let _r_ be ResolveLocale(%DurationFormat%.[[AvailableLocales]], _requestedLocales_, _opt_, %DurationFormat%.[[RelevantExtensionKeys]], %DurationFormat%.[[LocaleData]]).
         1. Let _locale_ be r.[[locale]].
         1. Set _durationFormat_.[[Locale]] to _locale_.
+        1. Set _durationFormat_.[[DataLocale]] to _r_.[[dataLocale]].
+        1. Let _dataLocale_ be _durationFormat_.[[DataLocale]].
+        1. Let _dataLocaleData_ be _durationFormat_.[[LocaleData]].[[&lt;_dataLocale_&gt;]].
```

Comment:
This doesn't look right?

Reply:
Also you no longer need `dataLocaleData` IIUC, you can directly extract the value of `digitalFormat` and set the values in the instance from that.

Reply:
going to leave this one for now -- I'm working on a PR to update DurationFormat to reflect changes made in https://github.com/tc39/ecma402/pull/849